packages/syft/src/syft/service/worker/image_registry_service.py

```diff
@@ -62,7 +62,7 @@ def delete(
         self,
         context: AuthedServiceContext,
         uid: UID = None,
-        url: str = None,
+        url: str = "",
```

Comment:
Check whether this should be `Optional[str]` or just `url: str` (without default value).

Reply:
This needs to be Optional[str]